_posts/2019-09-11-how-to-stay-up-to-date.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: How to stay up to date with your Rails application
+date: 2019-08-27 13:56:00
+categories: ["Rails", "Upgrade"]
+author: bronzdoc
+---
+
+How to stay up to date with your rails application
```

Comment:
@bronzdoc This should be an intro paragraph. Right now it is just repeating the title. You want to give a summary of what this article is about. It shows up in the articles index page and it should make the visitor want to read your article 😄